src/components/game/index.jsx

```diff
@@ -1,9 +1,66 @@
-import React from "react";
+import React, { useState } from "react";
+
 import Board from "components/game/Board";
 
 // Renders the Board component
 function Game() {
-  return <Board />;
+  const [valuesArr, setValuesArr] = useState(Array(9).fill(null)); // Values of the 1st-9th square
+  const [nextPlayer, setNextPlayer] = useState("O"); // Initial player aka first player to go is O
+  const winner = winnerChecker(valuesArr);
```

Comment:
`winnerChecker` gets computed every `rerender`. You can improve this by turning `winner` into a state and setting it every time `updateValuesArr` gets called. That way the state only changes/gets computed when necessary.